src/metabase/events/audit_log.clj

```diff
@@ -1,100 +1,93 @@
 (ns metabase.events.audit-log
   "This namespace is responsible for publishing events to the audit log. "
   (:require
-   [clojure.data :as data]
-   [metabase.api.common :as api]
    [metabase.events :as events]
    [metabase.models.audit-log :as audit-log]
+   [metabase.util :as u]
    [methodical.core :as methodical]
    [toucan2.core :as t2]))
 
 (derive ::event :metabase/event)
 
-(defn maybe-prepare-update-event-data
```

Comment:
Moved this logic into `metabase.models.audit-log` to make it a bit more flexible: now the `record-event!` params map can take `object` and `previous-object`, and it'll automatically record the changes if there are any